docs/migrations/v3-to-v4.md

```diff
@@ -18,6 +18,12 @@ No genesis or in-place migrations required when upgrading from v1 or v2 of ibc-g
 
 ## Chains
 
+### Fee Middleware
+
+The Fee Middleware module, as the name suggests, plays the role of an IBC middleware and as such must be configured by chain developers to route and handle IBC messages correctly.
+
+Please read the Fee Middleware [integration documentation](https://ibc.cosmos.network/main/middleware/ics29-fee/integration.html) for an in depth guide on how to congfigure the module correctly in order to incentivize IBC packets. 
```

Comment:
This looks good, but I don't know if we have any docs on actually integrating ics29 as an SDK module. Maybe we should just update the docs?

Reply:
I can make a follow up PR to update the docs as well?